docs/data/routes/docs/nextjs/getting-started/walkthrough-dotnetnew/en.md

```diff
@@ -3,4 +3,160 @@ name: walkthrough-dotnetnew
+
+To create a solution from the Getting Started template, you must complete the following procedures:
+
+- [Install the prerequisites](#)
+- [Install the template]()
+- [Create the MyProject solution]()
+
+### Install the prerequisites
+
+Before you can install the template and create a solution, you must make sure that you have these software components installed on your workstation:
+
+* [NodeJs 14.x](https://nodejs.org/) (we recommend using the latest LTS release.)
+
+* [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) (check your installed version with the `dotnet --version` command)
+
+* [.NET Framework 4.8 SDK ](https://dotnet.microsoft.com/download/dotnet-framework/net48) (see the [Microsoft procedure for checking .NET Framework versions](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed))
+
+* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+
+* [Docker for Windows](https://docs.docker.com/docker-for-windows/install/), with Windows Containers enabled
+
+See the [Sitecore Containers](https://containers.doc.sitecore.com/) documentation for more information on system requirements.
+
+### Install the template
+
+You download and install the template in your file system. Afterwards, you can create solutions based on the template.
+
+To install the template:
+
+1. Open PowerShell with administrator privileges.
+
+2. Run the following command:
+
+   ```powershell
+   dotnet new -i Sitecore.DevEx.Templates --nuget-source https://sitecore.myget.org/F/sc-packages/api/v3/index.json
+   ```
+
+   The `dotnet` tool does not provide obvious feedback that the install was successful. However, the install command ends by listing all of the installed templates. If the install is successful, the list includes `sitecore.aspnet.gettingstarted`.
+
+### Create the MyProject solution
+
+You can name you solution anything you like, but in this example we name the solution `MyProject`.
+
+To create the `MyProject` solution:
+
+1. Open PowerShell with administrator privileges.
+
+2. Check if you have Internet Information Server running on port 443:
+
+   ```powershell
+   Get-Process -Id (Get-NetTCPConnection -LocalPort 443).OwningProcess
+   ```
+
+   If you do, you must to stop it:
+
+   ```powershell
+   iisreset /stop
+   ```
+
+3. Check if you have Apache Solr or any other service running on port 8984:
+
+   ```powershell
+   Get-Process -Id (Get-NetTCPConnection -LocalPort 8984).OwningProcess
+   ```
+
+   If you do, you must stop it:
+
+   ```powershell
+   Stop-Service -Name "<the name of your service>"
+   ```
+
+   Or, if you have started it with Non-Sucking Service Manager, stop it with this command:
+
+   ```powershell
+   nssm stop "<the name of your service>"
+   ```
+
+4. Go to your solutions folder and create the `MyProject` solution and project:
+
+   ```powershell
+   dotnet new sitecore.nextjs.gettingstarted `
```

Comment:
As mentioned in Confluence, this can be simplified after release:

`dotnet new sitecore.nextjs.gettingstarted -n MyProject`